commons/src/main/java/com/powsybl/commons/reporter/LoggerReporter.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.reporter;
+
+import org.apache.commons.text.StringSubstitutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.*;
+import java.util.function.Consumer;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class LoggerReporter implements Reporter, ReportSeeker {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggerReporter.class);
+
+    private static final String DEFAULT_ROOT_TASK_KEY = "rootTaskKey";
+
+    private final String taskKey;
+    private final String defaultName;
+    private final LoggerReporter parentReporter;
```

Comment:
This parameter is not used yet, but could (should?) be used to construct a map containing all descendant reporters. This would allow to access directly any reporter from the root one.

Reply:
Parameter removed.